scala/scala.bzl

```diff
@@ -162,9 +162,31 @@ def _compile(ctx, cjars, dep_srcjars, buildijar):
     all_srcjars = set(srcjars + list(dep_srcjars))
     # look for any plugins:
     plugins = _collect_plugin_paths(ctx.attr.plugins)
+
+    if (hasattr(ctx.attr, 'enable_dependency_analyzer')
+        and ctx.attr.enable_dependency_analyzer
```

Comment:
the test will be to use the command line flags and see that we don't have the dependency analyzer.
But we'll need to wait for 0.5.3 or 0.6.0 for that change.
If you can add an internal issue about it and we'll add it later on